src/components/ui/carousel.tsx

```
@@ -177,7 +177,7 @@ const CarouselPrevious = React.forwardRef<HTMLButtonElement, React.ComponentProp
                 onClick={scrollPrev}
                 {...props}
             >
-                <ArrowLeft className="h-4 w-4" />
+                <ChevronLeft className="h-5 w-5" />
```

ESLint

[error] 'ChevronLeft' is not defined.

(react/jsx-no-undef)